.env.example

```diff
@@ -6,14 +6,26 @@ APP_LOG_LEVEL=20
 
 SERVICE_READINESS_TIMEOUT=60
 
-DB_SCHEME=postgresql
-DB_HOST=postgres
-DB_PORT=5432
-DB_USER=postgres
-DB_PASS=abc123
-DB_NAME=bancho
-DB_USE_SSL=false
-INITIALLY_AVAILABLE_DB=postgres
+READ_DB_SCHEME=postgresql
+READ_DB_HOST=postgres
+READ_DB_PORT=5432
+READ_DB_USER=postgres
+READ_DB_PASS=abc123
+READ_DB_NAME=bancho
+READ_DB_USE_SSL=false
+INITIALLY_AVAILABLE_READ_DB=postgres
+
+WRITE_DB_SCHEME=postgresql
+WRITE_DB_HOST=postgres
+WRITE_DB_PORT=5432
+WRITE_DB_USER=postgres
+WRITE_DB_PASS=abc123
+WRITE_DB_NAME=bancho
+WRITE_DB_USE_SSL=false
+INITIALLY_AVAILABLE_WRITE_DB=postgres
+
+DB_POOL_MIN_SIZE=2
+DB_POOL_MAX_SIZE=10
```

Comment:
this allows us to have separate read & write databases

typically known as database read replication